### ISSUE TITLE
MMS group users list missing

### ISSUE THREAD

**Issue by Tursko** [signalapp/Signal-Android#9725]
- [x] I have searched open and closed issues for duplicates
- [x] I am submitting a bug report for existing functionality that does not work as intended
- [x] I have read https://github.com/signalapp/Signal-Android/wiki/Submitting-useful-bug-reports
- [x] This isn't a feature request or a discussion topic

----------------------------------------

### Bug description
After the most recent group UI update, MMS groups no longer show a list of users. 

### Steps to reproduce
- Create an MMS group
- Look for user/phone number list

**Actual result:** List no longer exists.
**Expected result:** List of users/phone numbers should be present.

### Screenshots
![Screenshot_20200609-164656_2](https://user-images.githubusercontent.com/18274896/84216401-076f4780-aa8f-11ea-85e1-2d19dc76b87e.png)

### Device info

**Device:** Pixel 4
**Android version:** 10
**Signal version:** 4.62.4

**Comment by znafziger** [comment]
I am experiencing this issue as well. Very frustrating when you have a lot of groups with some of the same members. Especially when not everyone has actively participated in the chat.

**Comment by Tursko** [comment]
Yes this is for sure an issue. I have no clue who's in my chats now.
